src/scripts/clipperUI/components/sectionPicker.tsx

```diff
@@ -281,7 +281,7 @@ export class SectionPickerClass extends ComponentBase<SectionPickerState, Sectio
 		return (
 			<div id={Constants.Ids.locationPickerContainer}>
 				<div id={Constants.Ids.optionLabel} className="optionLabel">
-					<label className="buttonLabelFont" style={Localization.getFontFamilyAsStyle(Localization.FontFamily.Regular)}>
+					<label for="sectionLocationContainer" className="buttonLabelFont" style={Localization.getFontFamilyAsStyle(Localization.FontFamily.Regular)}>
```

Comment:
We should create a Constant for this one

Reply:
I'm considering this codebase practically abandoned, so I won't make changes for maintainability